README.md

```diff
@@ -1,2 +1,9 @@
 # blackduck-postgres
 Black Duck implementation of postgresql image for use with Black Duck application.
+
+## Database Creation
+
+Black Duck requires multiple databases within Postgres, which are created in one of two ways:
+
+- The main application database, `bds_hub`, is created via the `POSTGRES_DB` environment variable and behavior defined in the base postgres container
```

Comment:
I would add clarification that it is set inside Dockerfile .

Reply:
Added. Note: I do not have permission to merge this, even once approved